src/Features/CSharp/Portable/ConvertIfToSwitch/CSharpConvertIfToSwitchCodeRefactoringProvider.Rewriting.cs

```diff
@@ -77,7 +77,7 @@ public override SyntaxNode CreateSwitchStatement(IfStatementSyntax ifStatement,
                 closeParenToken: ifStatement.CloseParenToken.WithPrependedLeadingTrivia(ElasticMarker),
                 openBraceToken: block?.OpenBraceToken ?? Token(SyntaxKind.OpenBraceToken),
                 sections: List(sectionList.Cast<SwitchSectionSyntax>()),
-                closeBraceToken: block?.CloseBraceToken ?? Token(SyntaxKind.CloseBraceToken));
+                closeBraceToken: block?.CloseBraceToken.WithLeadingTrivia(SyntaxTriviaList.Empty) ?? Token(SyntaxKind.CloseBraceToken));
```

Comment:
I think we have WithoutLeadingTrivia

Reply:
I tried that first and it did not appear in intellisense. I'll try again.

Reply:
@CyrusNajmabadi It's defined on SyntaxNode only, not SyntaxToken. Should I create new extension methods?

Reply:
yup!

Reply:
Internal or public API?

Reply:
InternalsVisibleTo doesn't seem to be in place.

Reply:
Done.